NuGet.config

```diff
@@ -11,7 +11,7 @@
     <add key="vs-buildservices" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="myget-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
We should remove all dotnet.myget.org urls from the NuGet config. That account has been deactivated and deleted.

Reply:
Good catch. Its now removed.